plsql/ExcelGen.pkb

```diff
@@ -3255,6 +3286,9 @@ create or replace package body ExcelGen is
     ctx_cache(p_ctxId).sheetIndexMap(sd.sheetName) := sd.sheetIndex;
     
     return sd.sheetIndex;
+  exception when others then
```

Comment:
Same here.
I'd remove the exception handler and just trace the sheetIndex at the beginning of the procedure.